tests/test_flirt.py

```diff
@@ -0,0 +1,54 @@
+import vivisect.const as v_const
+from viv_utils.flirt import *
+
+from fixtures import sample_038476
+
+
+def test_invalid_function(sample_038476):
+    # this is an address that is not a function
+    assert is_only_called_from_library_functions(sample_038476, 0x400000) is False
+
+
+def test_function_not_called(sample_038476):
+    # this is a function that is not called by any other function
+    assert is_only_called_from_library_functions(sample_038476, 0x408155) is False
+
+
+def test_library_function(sample_038476):
+    # this is an existing library function
+    func_addr = 0x407660
```

Comment:
ugh, this is `main` which we probably should handle separately (same for `start` [the entry point function]?)

Reply:
this is TODO!